src/jdk.compiler/share/classes/com/sun/source/doctree/DocTree.java

```diff
@@ -38,241 +38,321 @@
         /**
          * Used for instances of {@link AttributeTree}
          * representing an attribute in an HTML element or tag.
+         *
+         * @since 1.8
```

Comment:
Do you need `@since` on members when the value is the same as for the top-level class?

Reply:
Right; when the @since for a particular member matches the @since for the top-level class, the @since on the member should be elided.

Reply:
I removed explicit `@since 1.8`; the top-level `@since 1.8` on com.sun.source.doctree.DocTree should suffice. We should consider adding a sentence or two on scope of `@since` to Documentation Comment Specification for the Standard Doclet.